src/explore-education-statistics-common/src/modules/charts/components/__tests__/MapBlockInternal.test.tsx

```diff
@@ -59,28 +59,28 @@ describe('MapBlockInternal', () => {
       // UK polygon
       expect(paths[0]).toHaveAttribute('fill', '#3388ff');
       // Location polygons
-      expect(paths[1]).toHaveAttribute('fill', '#5c80d6');
-      expect(paths[2]).toHaveAttribute('fill', '#86bcff');
-      expect(paths[3]).toHaveAttribute('fill', '#314573');
+      expect(paths[1]).toHaveAttribute('fill', 'rgba(145, 161, 201, 1)');
+      expect(paths[2]).toHaveAttribute('fill', 'rgba(218, 224, 237, 1)');
+      expect(paths[3]).toHaveAttribute('fill', 'rgba(71, 99, 165, 1)');
     });
 
     const legendItems = screen.getAllByTestId('mapBlock-legend-item');
 
     expect(legendItems).toHaveLength(5);
-    expect(legendItems[0]).toHaveTextContent('3.0% to 3.2%');
-    expect(legendItems[1]).toHaveTextContent('3.3% to 3.4%');
-    expect(legendItems[2]).toHaveTextContent('3.5% to 3.6%');
-    expect(legendItems[3]).toHaveTextContent('3.7% to 3.8%');
-    expect(legendItems[4]).toHaveTextContent('3.9% to 4.0%');
+    expect(legendItems[0]).toHaveTextContent('3.0% to 3.1%');
+    expect(legendItems[1]).toHaveTextContent('3.2% to 3.3%');
+    expect(legendItems[2]).toHaveTextContent('3.4% to 3.5%');
+    expect(legendItems[3]).toHaveTextContent('3.6% to 3.7%');
+    expect(legendItems[4]).toHaveTextContent('3.8% to 4.0%');
 
     const legendColours = screen.getAllByTestId('mapBlock-legend-colour');
 
     expect(legendColours).toHaveLength(5);
-    expect(legendColours[0].style.backgroundColor).toBe('rgb(134, 188, 255)');
-    expect(legendColours[1].style.backgroundColor).toBe('rgb(113, 158, 255)');
-    expect(legendColours[2].style.backgroundColor).toBe('rgb(92, 128, 214)');
-    expect(legendColours[3].style.backgroundColor).toBe('rgb(71, 99, 165)');
-    expect(legendColours[4].style.backgroundColor).toBe('rgb(49, 69, 115)');
+    expect(legendColours[0].style.backgroundColor).toBe('rgb(218, 224, 237)');
+    expect(legendColours[1].style.backgroundColor).toBe('rgb(181, 193, 219)');
+    expect(legendColours[2].style.backgroundColor).toBe('rgb(145, 161, 201)');
+    expect(legendColours[3].style.backgroundColor).toBe('rgb(108, 130, 183)');
+    expect(legendColours[4].style.backgroundColor).toBe('rgb(71, 99, 165)');
```

Comment:
Not related to this test specifically, but will these colour changes affect existing maps? If so, is that ok?

Reply:
Colour changes will indeed existing maps. I'd say this acceptable.